packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -9,15 +9,15 @@ import classnames from 'classnames';
 import { useViewportMatch, useMergeRefs } from '@wordpress/compose';
 import { forwardRef } from '@wordpress/element';
 import { useSelect } from '@wordpress/data';
-import {
-	getBlockType,
-	store as blocksStore,
-	__unstableGetInnerBlocksProps as getInnerBlocksProps,
-} from '@wordpress/blocks';
+import { getBlockType, store as blocksStore } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
+import { getExperimentalAPIs } from '../../experiments';
+const { __unstableGetInnerBlocksProps: getInnerBlocksProps } =
+	getExperimentalAPIs( '@wordpress/blocks' );
```

Comment:
Feels a bit weird to have an import from another package come under "Internal dependencies", but I don't have any immediate ideas as to how we might work around that 😅 - unless we add a new section e.g. "Experimental WP dependencies"?

Reply:
Agreed! Let's figure this out once we confirm this approach works for everyone.

Reply:
This can now be placed under the WordPress dependencies section – see https://github.com/WordPress/gutenberg/pull/43386#issuecomment-1257406628 for more details